pylint_odoo/test/main.py

```diff
@@ -142,9 +142,9 @@ def test_25_checks_without_coverage(self):
         # Some messages can be excluded as they are only applied on certain
         # Odoo versions (not necessarily 8.0).
         excluded_msgs = {
-            'xml-deprecated-qweb-directive',
+            'xml-attribute-translatable',
         }
-        extra_params = ['--valid_odoo_versions=8.0']
+        extra_params = []
```

Comment:
why you remove this extra params?

Reply:
Sorry, not this change is not necessary, I only adding the deprecated-data-xml-node and openerp-exception-warning because this check not run over odoo 8.0